tied-buffer/sycl-2.2/buffer-tags.md

```diff
@@ -75,6 +75,7 @@ namespace buffer {
   struct use_host_ptr;
   struct cl_interop;
   struct svm;  // SYCL 2.2 Only
+  struct mutex { mutex(mutex_class &mutexRef); };
```

Comment:
can you change the name of the property to "use_mutex" instead ? it avoids confusions with the C++ names